src/kube-runtime/GOPATH/src/github.com/microsoft/runtime/pkg/aggregator/error_aggregator.go

```diff
@@ -0,0 +1,461 @@
+package aggregator
+
+import (
+	"errors"
+	"io"
+	"io/ioutil"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/microsoft/runtime/pkg/logger"
+	"gopkg.in/yaml.v2"
+)
+
+// Use zero to present undefined exitCode in error spec
+const undefinedExitCode = 0
+
```

Comment:
0 is so confusing, why not use 256?

Reply:
Thanks, will fix

---
In reply to: [322048988](https://github.com/microsoft/pai/pull/3585#discussion_r322048988) [](ancestors = 322048988)